README.md

```diff
@@ -26,10 +26,18 @@ can't fallback to a O(n log² n) algorithm when there isn't enough extra heap me
   type such as `void`.
 
 
+Merging sorted ranges efficiently is an important part of the TimSort algorithm. This library exposes its merge
+algorithm in the public API. According to the benchmarks, `gfx::timmerge` is slower than `std::inplace_merge` on
+randomized sequences of simple elements, but it is faster for complex elements such as `std::string` and
+partially-sorted ranges. `gfx::timmerge` should be usable as a drop-in replacement for `std::inplace_merge`, with the
+difference that it can't fallback to a O(n log n) algorithm when there isn't enough extra heap memory available. Like
+`gfx::timsort`, `gfx::timmerge` can take a projection function and avoids using the postfix `++` or `--` operators.
```

Comment:
I don't know whether it's worth mentioning here: the `++`/`--` thing feels kind of obvious, and the projection support can be deduced from the function template declarations. It feels a bit redundant.

Reply:
Yes, the reader of the README can figure out one of the facts by examining the signatures below and the other - by reading the implementation of `timmerge`. However, relying on the current implementation is not the same as relying on the documentation. If the README doesn't state the `++`/`--` guarantee, then a future version of `timmerge` might break the user code. This one sentence is pretty short, so removing it won't noticeably reduce the length of the README.